src/app/date-picker/date-picker.component.html

```diff
@@ -19,6 +19,7 @@
          [attr.data-hidden]="!_areCalendarsShown">
       <dp-day-calendar #dayCalendar
                        *ngSwitchCase="'day'"
+                       [isMonthFirst]="isMonthFirst"
```

Comment:
I think that the name shouldn't be that strict, maybe one would want to start with a year. Let's change it to something more generic like enum